hardware/raspberrypi/revision-codes/README.md

```diff
@@ -86,6 +92,9 @@ uuuuuuuuFMMMCCCCPPPPTTTTTTTTRRRR
 |          |              | 11: 4B                     |
 | RRRR     | Revision     | 0, 1, 2, etc.              |
 
+<sup>1</sup> Information on programming the OTP bits can be found [here](../../industrial/README.md) and [here](../otpbits.md)
```

Comment:
Should this also link to [configuration/config-txt/overclocking.md](https://github.com/raspberrypi/documentation/blob/master/configuration/config-txt/overclocking.md) , where `never_over_voltage` is documented?

Reply:
This link has now been added in the documentation of the warranty bit